dactyl-config.yml

```diff
@@ -2385,6 +2385,16 @@ pages:
         targets:
             - en
 
+    -   md: tutorials/manage-the-rippled-server/configuration/configure-statsd.md
+        html: configure-statsd.html
+        funnel: Docs
+        doc_type: Tutorials
+        category: Manage the rippled Server
+        subcategory: Configuration
+        blurb: Monitor your rippled server with StatsD metrics. #TODO:translate
+        targets:
+            - en
```

Comment:
Add this to the `- ja` target as well since we don't have a translated version for now.

Reply:
Thanks @mDuo13. I made the changes you requested in my latest commit.